internal/store/driver_test.go

```diff
@@ -278,8 +278,56 @@ var _ = Describe("Driver", func() {
 
 			status := driver.calculateIngressLoadBalancerIPStatus(&i1, c)
 			Expect(len(status)).To(Equal(2))
-			Expect(status[0].Hostname).To(Equal(cname1))
-			Expect(status[1].Hostname).To(Equal(cname2))
+			Expect(status[0].Hostname).To(Not(Equal(status[1].Hostname)))
```

Comment:
How do you feel about:

```go
hostnames := make([]string, len(status))
for i, s := range status {
        hostnames[i] = s.Hostname
}
Expect(hostnames).To(Equal([]string{cname1, cname2}))

```

Reply:
I read through https://github.com/onsi/gomega/blob/master/matchers.go and found a way to avoid the loop, let me know what you think